lib/ts/BaseLogFile.cc

```diff
@@ -27,19 +27,30 @@
  * This consturctor creates a BaseLogFile based on a given name.
  * This is the most common way BaseLogFiles are created.
  */
-BaseLogFile::BaseLogFile(const char *name) : m_signature(0), m_has_signature(false)
+BaseLogFile::BaseLogFile(const char *name)
+  : m_fp(nullptr),
```

Comment:
Since you are around this code, why not use the C++11 style initialization (when possible), in the declaration of the member variables? I personally find that a lot easier than the old style. However, I'm nog going to block this PR because of this :).